index.html

```diff
@@ -46,22 +46,22 @@ <h3>My Skills</h3>
       <section class="interests">
         <article>
           <h2>Interests</h2>
-          <ul>
-            <li>Coden lernen</li>
-            <li>Videogames</li>
-            <li>Lorem ipsum</li>
-            <li>Lorem ipsum</li>
+          <ul class="interests-list">
+            <li class="interests-list-item">Coden lernen</li>
+            <li class="interests-list-item">Videogames</li>
+            <li class="interests-list-item">Lorem ipsum</li>
+            <li class="interests-list-item">Lorem ipsum</li>
           </ul>
         </article>
         <article>
           <h2>Skills</h2>
-          <ul>
-            <li>HTML</li>
-            <li>CSS</li>
-            <li>JavaScript</li>
-            <li>React</li>
-            <li>Git & GitHub</li>
-            <li>MongoDB</li>
+          <ul class="skills-list">
+            <li class="skills-list-item">HTML</li>
+            <li class="skills-list-item">CSS</li>
+            <li class="skills-list-item">JavaScript</li>
+            <li class="skills-list-item">React</li>
+            <li class="skills-list-item">Git & GitHub</li>
+            <li class="skills-list-item">MongoDB</li>
```

Comment:
Instead of assigning the same class to all list items you could select and style the `<li>` tags like this:

```css
.skills-list > li {
   /* Do your styling here*/ 
}
```
This would make your HTML code a little bit leaner...